src/main/resources/data/pieces/52590_default.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "e67dbaaf-911b-1024-b083-8d92af285670",
-  "caption": "Tutorial Volume 6",
+  "displaySummary": "Tutorial Volume 6",
```

Comment:
seems we need more updates in this repo:
![image](https://github.com/folio-org/mod-orders-storage/assets/25097693/7b961a81-b93c-4c32-b2ee-914e5c052d0b)
also based on UI requirements we need to do this renaming also in receiving history: https://issues.folio.org/browse/UIIN-2740
Also because it is bracking change - we need to increase major version of pieces interface in module_descriptor

Reply:
Are we updating checkinCollection.json and receiving_history.json data types?

Reply:
Yes, based on requirements from UIIN-2740 we need to update receiving_history.json
I thinks checkingCollection should be updated as well to not have confusion between field names.
Also major versions od both Checkin/Recieve and Receiving History APIs should be increased in Module_descriptor_template

Reply:
Done. Changes related to mod-orders moved to PR for https://issues.folio.org/browse/MODORDERS-1010